### PR TITLE
Add SPARTN and Septentrio SBF

### PR DESCRIPTION
**Pull request by PaulZC**
SBF has helper methods and an optional callback for invalid data
See the new SBF_in_SPARTN_Test example
Enjoy!

### CODE REVIEW

File: examples/SBF_Test/SBF_TEST.ino [hunk 1]
```diff
@@ -0,0 +1,208 @@
+/*
+  SparkFun SBF test example sketch
+
```
Comment (nseidle) [inline examples/SBF_Test/SBF_TEST.ino:3]: What is this sketch demonstrating or proving?

File: examples/SPARTN_Test/SPARTN_TEST.ino [hunk 2]
```diff
@@ -0,0 +1,261 @@
+/*
+  SparkFun SPARTN test example sketch
+
```
Comment (nseidle) [inline examples/SPARTN_Test/SPARTN_TEST.ino:3]: What's this sketch do?

**Comment by nseidle** [comment]
Looks good to me. Please add a couple comments to the example INOs and I'll merge.

**Comment by PaulZC** [comment]
@nseidle : Done! I added comments to the existing earlier examples too... ;-)

**Pull request merged**